### PR TITLE
WebSockets: saving data to redis/postgres

### PR DESCRIPTION
**Pull request by SpennyNDaJets**
## Summary
WebSocket events are getting sent to the backend, but just dropped. This adds functionality to storage the WebSocket events in Redis and S3.

## TODOs
- [x] Handle large payloads -> leave for now, and come back to see if there is an issue when loading into JS
- [x] Handle empty payloads -> leave these in for now
- [x] What versions to bump

## How did you test this change?
**Click test:**
Before: Check out [this PR](https://github.com/highlight/highlight/pull/5727) that has the storage changes, along with some frontend changes to better test
1) Create a new session that watches a live session (i.e. creates web socket connections)
2) Exit the session and let it process
3) Find the `GetSession` graphql query, and find the `web_socket_events_url`
4) Click on the `web_socket_events_url` to view the events in your browser
- [x] Events are correct captured and stored

https://www.loom.com/share/66c5f3ed87a14d55b2b81838ebc151ce?sid=7fec7489-7a76-4fed-93cd-0f68aa871c96

## Are there any deployment considerations?
There should not be any events recorded to S3 since the new npm package has not been built yet - watch S3 storage and errors around payload being too large.

### CODE REVIEW

File: sdk/client/src/listeners/network-listener/utils/models.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export interface RequestResponsePair {
 
 export interface WebSocketRequest {
 	socketId: string
-	initiatorType: 'WebSocket'
```
Comment (SpennyNDaJets) [inline sdk/client/src/listeners/network-listener/utils/models.ts:30]: Keep consistent with the lowercase initiatorType of other requests

**Review by mayberryzane — APPROVED** [review]

**Comment by SpennyNDaJets** [comment]
Tagging @Vadman97 for recommendations on what versions should be bumped

**Review by Vadman97 — COMMENTED** [review]
sorry for the delay, had realized i wrote some comments but forgot to submit the review :(

**Comment by Vadman97** [comment]
> Tagging @Vadman97 for recommendations on what versions should be bumped

@SpennyNDaJets just `sdk/firstload/package.json` with a minor version bump if this is backwards compatible (updating `sdk/firstload/CHANGELOG.md` as well).

**Review by Vadman97 — APPROVED** [review]
nice! looks good @SpennyNDaJets. bump @mayberryzane regarding the `AvoidPostgresStorage` removal

**Comment by mayberryzane** [comment]
> bump @mayberryzane regarding the AvoidPostgresStorage removal
Sorry didn't notice this comment earlier, we're good to delete this!

**Pull request merged**